lib/routes/account.js

```diff
@@ -544,19 +541,26 @@ module.exports = function (
         }
 
         function sendVerifyAccountEmail() {
-          // For legacy clients, behavior is to not send an email to verify their account
-          // and have the requestor send it.
-          var sendEmailIfUnverified = false
-
-          // If a value was passed, use that instead
-          if (request.payload.sendEmailIfUnverified !== undefined) {
-            sendEmailIfUnverified = request.payload.sendEmailIfUnverified
+          // Delegate sending emails for unverified users to auth-server.
```

Comment:
There is a lot of logic to obtain `shouldSendVerifyAccountEmail` - think it's worthy of extracting into a helper fucntion that accepts a `request`?